src/ios/TwilioVideoPlugin.m

```diff
@@ -14,10 +14,7 @@ - (void)openRoom:(CDVInvokedUrlCommand*)command {
     NSArray *args = command.arguments;
     NSString* token = args[0];
     NSString* room = args[1];
-    TwilioVideoConfig *config = [[TwilioVideoConfig alloc] init];
-    if ([args count] > 2) {
-        [config parse: command.arguments[2]];
-    }
+    TwilioVideoConfig *config = [TwilioVideoConfig configFromDict:command.arguments[2]];
```

Comment:
I added a new call to create the config, so we always make sure the config is initialized properly. Some missing fields where causing crashes when the video connection failed.